code/datums/elements/tear_wall.dm

```diff
@@ -1,29 +1,31 @@
 /**
  * Attached to a basic mob that will then be able to tear down a wall after some time.
  */
-/datum/component/tear_wall
+/datum/element/tear_wall
+	element_flags = ELEMENT_BESPOKE
+	argument_hash_start_idx = 3
 	/// The rate at which we can break regular walls
-	var/regular_tear_time = 2 SECONDS
+	var/regular_tear_time
 	/// The rate at which we can break reinforced walls
-	var/reinforced_tear_time = 4 SECONDS
+	var/reinforced_tear_time
 	/// If we are already deconstructing a wall
 	var/tearing_wall = FALSE
```

Comment:
dang, I hadn't noticed this variable. This should be turned into a trait added to the mob tearing the wall, otherwise only one mob can ever tear a wall at a time.